doc/doxygen/headers/boundary.h

```diff
@@ -1,131 +0,0 @@
-// ---------------------------------------------------------------------
-//
-// Copyright (C) 2003 - 2018 by the deal.II authors
-//
-// This file is part of the deal.II library.
-//
-// The deal.II library is free software; you can use it, redistribute
-// it, and/or modify it under the terms of the GNU Lesser General
-// Public License as published by the Free Software Foundation; either
-// version 2.1 of the License, or (at your option) any later version.
-// The full text of the license can be found in the file LICENSE at
-// the top level of the deal.II distribution.
-//
-// ---------------------------------------------------------------------
-
-
-/**
- * @defgroup boundary Boundary and manifold description for triangulations
```

Comment:
was there anything worth saving from this file?

Reply:
OK. So then this is ready to merge, right? We can handle the glossary entry in a separate patch.